frontend/.eslintrc.json

```
@@ -1,3 +1,27 @@
 {
-  "extends": "next/core-web-vitals"
-}
+  "plugins": [
+    "@typescript-eslint"
+  ],
+  "extends": [
+    "next/core-web-vitals",
+    "prettier",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "overrides": [
+    {
+      "files": [
+        "*.ts",
+        "*.tsx"
+      ],
+      "parserOptions": {
+        "project": [
+          "./tsconfig.json"
+        ]
+      }
+    }
+  ],
+  "rules": {
```

Maybe we should also disable rule with  no-explicit-any. I know it is the right thing to avoid using any, but i don't think at any cost we should always do this. sometimes we can got into troubles with that and i think we will have to even add ticket for resolving issues with that.
```suggestion
  "rules": {
  "@typescript-eslint/no-explicit-any": "off",
   "eslint-plugin-import/extensions": "off"
  }
}
```

---

Yep, I agree